specification/professionalservice/resource-manager/Microsoft.ProfessionalService/preview/2023-07-01-preview/professionalservice.json

```diff
@@ -0,0 +1,1038 @@
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "format": "int32",
+                "type": "integer"
+              }
+            }
+          },
+          "204": {
+            "description": "Unsubscribe operation completed successfully."
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Unsubscribe Subscription Level ProfessionalService resource": {
+            "$ref": "./examples/ProfessionalService/ProfessionalServiceUnsubscribe.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources": {
+      "post": {
+        "tags": [
+          "SubscriptionLevelProfessionalServiceResources"
+        ],
+        "operationId": "ProfessionalServiceSubscriptionLevel_ValidateMoveResources",
+        "description": "Validate whether a specified Subscription Level ProfessionalService can be moved.",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "moveResourceParameter",
+            "in": "body",
+            "description": "Object that represents the resources to move.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/moveResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Validation was successful and resources can be moved"
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Validate move of a subscription Level ProfessionalService resource": {
+            "$ref": "./examples/ProfessionalService/ValidateResourceMove.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/moveResources": {
```

Comment:
Same here

Reply:
This follows the standard mentioned at https://armwiki.azurewebsites.net/api_contracts/ResourceMove.html